tests/h/views/api/annotations_test.py

```diff
@@ -81,96 +82,62 @@ def search_run(self, search_lib):
     "storage",
 )
 class TestCreate:
```

Comment:
These tests we're of the overly split up kind where every assertion was in a different test. In some cases we have two tests for a single line of code.

I've bunched these up into fewer cases up top, with failure cases below.

The same is true later in the file.